docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx

```diff
@@ -0,0 +1,98 @@
+---
+slug: /manuals/administrator/engine/472258/custom-runner
+---
+
+# Use a Custom Runner
```

Comment:
@sipsma this is derived from https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md. Could you please review and let me know if any changes are needed?